runhouse/main.py

```diff
@@ -121,10 +124,18 @@ def _start_server(restart, restart_ray, screen, create_logfile=True):
 def start(
     restart_ray: bool = typer.Option(False, help="Restart the Ray runtime"),
     screen: bool = typer.Option(False, help="Start the server in a screen"),
+    custom_server_host: Optional[str] = typer.Option(
```

Comment:
This is code golf but I think I'd prefer for this to just be `host`

Reply:
Done